config/stack/manifests/app.yaml

```diff
@@ -2,7 +2,7 @@ title: "Minimal GCP Environment"
 readme: ""
 overview: "This stack provisions a private network and creates resource classes that has minimal node settings and refer to that private network."
 overviewShort: "Start using GCP with Crossplane without needing to create your own resource classes!"
-version: 1.0
+version: 0.0.1
```

Comment:
I believe we haven't published any version of this Stack, so it is safe to (and better to) reset this to 0.0.1, as you have done here.

Reply:
I believe we started other stacks at `0.1.0`. Not saying this is necessarily wrong, but just something to keep in mind. If we go `0.0.1` then any subsequent patch release will be somewhat confusing.

Reply:
Are you suggesting `0.1.0`? That's fine with me.

We noticed that `1.0` seemed like a bogus semver and we didn't want to make this `1.0.1`, since this hasn't been released and doesn't represent that level of quality, so we reset the version to the lowest value imaginable, + 0.0.1

Reply:
I just looked at the other ones just for the heck of it and oddly enough those versions in the app files have never been updated from `0.0.1`. I am not sure if that was a mistake or not.

Either way I am fine changing this to whatever it _should_ be and for the ui-schema file as well.

Reply:
I am in favor of `v0.1.0` per semantic versioning spec: https://semver.org/

Reply:
Oh, https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase recommends 0.1.0 as a starting point, although, [as others observe](https://stackoverflow.com/a/38256419/156674), they don't preclude 0.0.1.  I wasn't aware of that.  Nice find.

Reply:
@rathpc and I chatted and I was unaware that this stack would be released in the same manner as the others. Since that is the case, we can actually just bump the version to be correct when we cut each release branch (this is what other stacks do as well). Leaving as `v0.0.1` is a nice reminder that it needs to be updated 👍

Reply:
> I just looked at the other ones just for the heck of it and oddly enough those versions in the app files have never been updated from 0.0.1. I am not sure if that was a mistake or not.

in the master branches you are correct @rathpc that the other stacks have their app.yaml version field still set to 0.0.1. I've actually been thinking about updating all those to `master` or something like that.  In the release branches though, they are all set to a semver that matches the release, e.g.: https://github.com/crossplaneio/stack-aws/blob/release-0.4/config/stack/manifests/app.yaml#L24

i'm good with just putting this as `0.0.1` in master, similar to all the other stacks.  in a release branch we can update it as it needs and is consistent with our current (however non ideal) process with the other stacks.